virtool/users/data.py

```diff
@@ -219,9 +229,17 @@ async def update(self, user_id: str, data: UpdateUserRequest):
                 permissions,
             )
 
-        user = await fetch_complete_user(self._authorization_client, self._mongo, self._pg, user_id)
+        user = await fetch_complete_user(
+            self._authorization_client, self._mongo, self._pg, user_id
+        )
 
         if user is None:
             raise ResourceNotFoundError
 
         return user
+
+    async def check_no_users_exist(self) -> bool:
+        """
+        Checks that no users exist.
+        """
+        return await self._mongo.users.count_documents({}, limit=1) == 0
```

Comment:
```suggestion
    async def check_users_exist(self) -> bool:
        """
        Checks that no users exist.
        """
        return await self._mongo.users.count_documents({}, limit=1) > 0
```
I would rework this to be a check if users exist, then `not` the returned value as needed where called. As written when checking for the first user the logic reads as " if not no users exist" which is a harder to grok than simply just checking if users exist"